python/ray/data/dataset.py

```diff
@@ -209,7 +209,7 @@ class Dataset(Generic[T]):
         >>> ds.map_batches(lambda batch: [v * 2 for v in batch])
         MapBatches(<lambda>)
         +- Dataset(num_blocks=17, num_rows=1000, schema=<class 'int'>)
-        >>> # Compute max.
+        >>> # Compute maximum
```

Comment:
super nit: let's keep `.`, thanks.

Reply:
Can we merge this as is? I don't want to wait for the tests to pass again. This is kind of blocking visibility into RLlib ci failures. Alternatively I can add the dot but don't wait for the tests to pass.

Reply:
Sure, go ahead.